### PR TITLE
Add ETA to unlockWithdraw

### LINKED ISSUE #885: Add ETA for unlockWithdraw

### PR DESCRIPTION
**Pull request by shyam-patel-kira**
fixes #885

### CODE REVIEW

File: cmd/unlockWithdraw.go [hunk 1]
```diff
@@ -80,19 +80,31 @@ func (*UtilsStruct) HandleWithdrawLock(client *ethclient.Client, account types.A
...
+		if waitFor.Cmp(big.NewInt(1)) == 0 {
+			log.Infof("Withdrawal period not reached. Cannot withdraw now, please wait for %d epoch! (approximately %s)", waitFor, razorUtils.SecondsToReadableTime(int(timeRemaining)))
+		} else {
+			log.Infof("Withdrawal period not reached. Cannot withdraw now, please wait for %d epochs! (approximately %s)", waitFor, razorUtils.SecondsToReadableTime(int(timeRemaining)))
 		}
-		txnOpts := razorUtils.GetTxnOpts(txnArgs)
+		return core.NilHash, nil
+	}
+
+	txnArgs := types.TransactionOptions{
```
Comment (shekhar2807) [inline cmd/unlockWithdraw.go:94]: You can move txnArgs inside the below if. It would be more efficient.

**Review by shekhar2807 — CHANGES_REQUESTED** [review]

File: cmd/unlockWithdraw.go (hunk 1 above)
Comment (shyam-patel-kira) [reply cmd/unlockWithdraw.go]: Done. Thanks

**Review by shekhar2807 — APPROVED** [review]

File: cmd/initiateWithdraw.go [hunk 2]
```diff
@@ -104,9 +104,9 @@ func (*UtilsStruct) HandleUnstakeLock(client *ethclient.Client, account types.Ac
 	if waitFor.Cmp(big.NewInt(0)) > 0 {
 		timeRemaining := uint64(waitFor.Int64()) * core.EpochLength
 		if waitFor.Cmp(big.NewInt(1)) == 0 {
-			log.Infof("Withdrawal period not reached. Cannot withdraw now, please wait for %d epoch! (approximately %s)", waitFor, razorUtils.SecondsToReadableTime(int(timeRemaining)))
+			log.Infof("Withdrawal Initiation period not reached. Cannot initiate withdraw now, please wait for %d epoch! (approximately %s)", waitFor, razorUtils.SecondsToReadableTime(int(timeRemaining)))
```
Comment (ashish10677) [inline cmd/initiateWithdraw.go:107]: Remove this if else, keep one message.
`"Withdrawal Initiation period not reached. Cannot initiate withdraw now, please wait for %d epoch(s)! (approximately %s)`

File: cmd/unlockWithdraw.go [hunk 3]
```diff
@@ -80,6 +80,17 @@ func (*UtilsStruct) HandleWithdrawLock(client *ethclient.Client, account types.A
 		return core.NilHash, err
 	}
 
+	waitFor := big.NewInt(0).Sub(withdrawLock.UnlockAfter, big.NewInt(int64(epoch)))
+	if waitFor.Cmp(big.NewInt(0)) > 0 {
+		timeRemaining := uint64(waitFor.Int64()) * core.EpochLength
+		if waitFor.Cmp(big.NewInt(1)) == 0 {
+			log.Infof("Withdrawal period not reached. Cannot withdraw now, please wait for %d epoch! (approximately %s)", waitFor, razorUtils.SecondsToReadableTime(int(timeRemaining)))
```
Comment (ashish10677) [inline cmd/unlockWithdraw.go:87]: Remove if else here as well.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: cmd/initiateWithdraw.go (hunk 2 above)
Comment (shyam-patel-kira) [reply cmd/initiateWithdraw.go]: Done

File: cmd/unlockWithdraw.go (hunk 3 above)
Comment (shyam-patel-kira) [reply cmd/unlockWithdraw.go]: Removed

**Review by ashish10677 — APPROVED** [review]

**Pull request merged**